### PR TITLE
Type check amp-app-banner

### PR DESCRIPTION
**Pull request by mkhatib**

### CODE REVIEW

File: extensions/amp-app-banner/0.1/amp-app-banner.js [hunk 1]
```diff
@@ -38,6 +38,17 @@ const TAG = 'amp-app-banner';
  */
 export class AbstractAppBanner extends AMP.BaseElement {
```
Comment (erwinmombay) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: lets also add `@abstract` annotation here. (recently added)

Comment (mkhatib) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: Done!

**Comment by mkhatib** [comment]
PTAL 👀

**Comment by erwinmombay** [comment]
LGTM! 👍

**Pull request merged**